src/components/Reports/LocationSection.tsx

```diff
@@ -224,9 +226,11 @@ export default function LocationSection(props: LocationSectionProps): JSX.Elemen
               value={(location as ReportNursery).capacity ?? ''}
               minNum={0}
               editable={editable}
-              onChange={(value) => onUpdateLocation('capacity', Math.floor(value as number))}
+              onChange={(value) =>
+                onUpdateLocation('capacity', value === '' ? null : Math.max(0, Math.floor(value as number)))
```

Comment:
Setting to null allows the field to be cleared, otherwise it gets stuck with a zero in the field

Reply:
@constanzauanini had a related fix i think, in web-comp

Reply:
looks like this could be extracted out into a function and reused in other places?
func(value) => value === '' ? null : Math.max(...)

Reply:
yeah, looks like I have quite a lot of duplication. I can definitely extract the function